src/textures/CubeTexture.js

```diff
@@ -39,20 +46,20 @@ Object.defineProperty( CubeTexture.prototype, 'images', {
 
 CubeTexture.prototype.fromEquirectangular = function( renderer, source, size ) {
 
-	var scene = new THREE.Scene();
+	var scene = new Scene();
 
 	var gl = renderer.getContext();
 	var maxSize = gl.getParameter( gl.MAX_CUBE_MAP_TEXTURE_SIZE )
 
-	var camera = new THREE.CubeCamera( 1, 100000, Math.min( size, maxSize ) );
+	var camera = new CubeCamera( 1, 100000, Math.min( size, maxSize ) );
 
-	var material = new THREE.MeshBasicMaterial( {
+	var material = new MeshBasicMaterial( {
 		map: source,
-		side: THREE.BackSide
+		side: BackSide
 	} );
 
-	var mesh = new THREE.Mesh(
-		new THREE.IcosahedronGeometry( 100, 4 ),
+	var mesh = new Mesh(
+		new IcosahedronGeometry( 100, 4 ),
```

Comment:
>Isn't that something that should be fixed in IcosahedronGeometry?

I do not believe there is a fix -- at least for `IcosahedronGeometry` having detail 0.

For higher details, it appears `geometry.rotateY( rad )` would work, for some value `rad`.

See http://threejs.org/examples/misc_uv_tests.html and hack in this code:
```javascript
test('new THREE.IcosahedronGeometry( 30, 0 )', new THREE.IcosahedronGeometry( 30, 0 ));
test('new THREE.IcosahedronGeometry( 30, 1 )', new THREE.IcosahedronGeometry( 30, 1 ));
test('new THREE.IcosahedronGeometry( 30, 2 )', new THREE.IcosahedronGeometry( 30, 2 ));
test('new THREE.IcosahedronGeometry( 30, 3 )', new THREE.IcosahedronGeometry( 30, 3 ));
test('new THREE.IcosahedronGeometry( 30, 4 )', new THREE.IcosahedronGeometry( 30, 4 ));
```

Also, I misspoke, all that should be required is
```javascript
texture.wrapS = THREE.RepeatWrapping;
```

`detail` of 2 should be sufficient, but yes, it is just a single render.